### PR TITLE
IGAPP-725: Language not set correctly on page (re)load

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Review by f1sh1918 — APPROVED** [review]

File: web/src/index.ejs [hunk 1]
```diff
@@ -36,21 +36,33 @@
...
 
+    <!-- Open Sans -->
     <link rel="preload"
           href="/fonts/open-sans/open-sans-v15-greek_cyrillic-ext_vietnamese_latin_cyrillic_greek-ext-700.woff2"
           as="font"
           type="font/woff2" crossorigin />
     <link rel="preload"
           href="/fonts/open-sans/open-sans-v15-greek_cyrillic-ext_vietnamese_latin_cyrillic_greek-ext-regular.woff2"
           as="font" type="font/woff2" crossorigin />
+    <link href='/fonts/open-sans/open-sans.css' rel='stylesheet' />
```
Comment (SteffiStuffel) [inline web/src/index.ejs:47]: Nice :tada:

**Review by SteffiStuffel — APPROVED** [review]
Code looks fine.

I tested this and the computed styles tab, says 
```Lateef—Network resource(12 glyphs)
Raleway—Network resource(1 glyph)```

**Comment by SteffiStuffel** [comment]
Bildschirmfoto zu 2021-08-04 07-55-13

**Comment by klinzo** [comment]
> Bildschirmfoto zu 2021-08-04 07-55-13

I guess this is because just the space is displayed in Raleway, everything else is Lateef as you are on arabic, right?

**Comment by SteffiStuffel** [comment]
> @SteffiStuffel I guess this is because just the space is displayed in Raleway, everything else is Lateef as you are on arabic, right?

I guess so? Could not find out which char it is, but looks fine.

**Pull request merged**